public/js/views/management/pay-methods.jsx

```diff
@@ -20,6 +21,10 @@ export default class PayMethods extends Component {
 
   handleDelPayMethod = e => {
     e.preventDefault();
+    if (isDisabled(e.target)) {
```

Comment:
sounds good. We'll just have to watch out for it in code reviews which is no biggie.